Doppler.ContactPolicies.Business.Logic/UserApiClient/Services/UserFeaturesService.cs

```diff
@@ -0,0 +1,43 @@
+using Doppler.ContactPolicies.Business.Logic.DTO;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Doppler.ContactPolicies.Business.Logic.UserApiClient.Services
+{
+    public class UserFeaturesService : IUserFeaturesService
+    {
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly IConfiguration _configuration;
+        private readonly ILogger<UserFeaturesService> _logger;
+
+        public UserFeaturesService(IHttpClientFactory clientFactory, IConfiguration configuration, ILogger<UserFeaturesService> logger)
```

Comment:
What do you think about a specific configuration like we are using on [this](https://github.com/FromDoppler/doppler-contact-policies-api/blob/059d5e43aa13e5f15e8a7c7ed49d21651e3caba6/Doppler.ContactPolicies.Data.Access/Core/DataBaseConnectionFactory.cs#L16) place? 
```suggestion
        public UserFeaturesService(IHttpClientFactory clientFactory, IOptions<UserFeaturesServiceSettings> userFeaturesServiceSettings, ILogger<UserFeaturesService> logger)
```